src/member-api.js

```diff
@@ -155,14 +155,21 @@ export class MemberApi extends TypedEmitter {
       const projectInviteId = projectKeyToProjectInviteId(this.#projectKey)
       const project = await this.#dataTypes.project.getByDocId(projectId)
       const projectName = project.name
+
+      // TODO: Require projectColor to be set as well?
```

Comment:
highlighting as it's related to my question about whether to make the `projectColor` field in an invite required or not

Reply:
Similar to other comments, I feel slightly uncomfortable with committing to a default color to use at the core level.

UX tradeoffs I'm thinking about in regards to clients with pre-existing projects that upgrade to this new API:

1. If we require color to be set, they'll receive an error about missing a color despite having been able to send invites before. The UI would need to expose a way to set this color (think that's already the plan).

2. Otherwise, we'll implicitly use the default color when sending new invites (until they explicitly set the color).

(1) feels more correct to me despite it seeming more inconvenient, but curious to get additional thoughts.

Reply:
But yeah I think making the color required here generally makes sense

Reply:
IMO it'd be better to have it optional for backwards compat and have the UI set a default if one is lacking. 🤷 Eventually it won't matter once everyone is updated.

Reply:
based on discussion, decided to not require project color here